src/fides/api/service/storage/storage_uploader_service.py

```diff
@@ -85,13 +98,18 @@ def _s3_uploader(
     auth_method = config.details[StorageDetails.AUTH_METHOD.value]
 
     return upload_to_s3(
-        config.secrets, data, bucket_name, file_key, config.format.value, privacy_request, auth_method  # type: ignore
+        config.secrets, data, bucket_name, file_key, config.format.value, privacy_request, auth_method, data_category_field_mapping  # type: ignore
```

Comment:
should we pass on the data_use_map too to `upload_to_s3`?

Reply:
yup, good catch! missed that.